conans/client/installer.py

```diff
@@ -31,7 +32,7 @@ def _init_package_info(deps_graph, paths, current_path):
             conan_file.package_folder = package_folder
             conan_file.cpp_info = CppInfo(package_folder)
         else:
-            conan_file.cpp_info = CppInfo(current_path)
+            conan_file.cpp_info = CppInfo("")
```

Comment:
Consumer project doesn't have a "package_folder", it has no sense, because by definition no other package will inherit the consumer cppinfo, so, why we need the path?